src/sass/emma-woocommerce.scss

```diff
@@ -0,0 +1,16 @@
+/*--------------------------------------------------------------
+# Emma - Theme Styles
```

Comment:
This comment should be `Emma - WooCommerce Styles`

Reply:
hanlded in d8bc22e61094f0ed3da2e85fa890fbfb6b504fd6